processing/data_collection/gazette/spiders/rn_mossoro.py

```diff
@@ -0,0 +1,63 @@
+from datetime import datetime, date
+
+import dateutil
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RnMossoroSpider(BaseGazetteSpider):
+
+    name = "rn_mossoro"
+    allowed_domains = ["jom.prefeiturademossoro.com.br"]
+
+    TERRITORY_ID = "2408003"
+
+    def __init__(self, start_date=None, end_date=None, *args, **kwargs):
+        self.start_date = date.today().replace(day=1)
+        self.end_date = date.today()
+
+        super(RnMossoroSpider, self).__init__(start_date, end_date, *args, **kwargs)
+
+        self.logger.debug(
+            "Start date is {date}".format(date=self.start_date.isoformat())
+        )
+        self.logger.debug("End date is {date}".format(date=self.end_date.isoformat()))
```

Comment:
The `end_date` attribute doesn't fit the current project pattern (although I like it and encourage discussion about it). Having a particular spider with a different set of parameters should only be done if absolutely necessary in my opinion.

Also, I don't think the logs add much. So right now I'm thinking this could become cleaner if the `__init__` method was removed. What do you think?

`start_date` as a class attribute and `end_date` as a variable in `start_requests` or something like that?

Reply:
It's been used on 3 or 4 other spiders I've submitted, and other reviewers seem to approve it.
Also, the logs are for debugging, they won't show up on "production" logs

Reply:
I just felt like it would make it more readable without losing important functionality. If you see advantage in it, there's no problem.

And for the `end_date` argument, nothing about this would block you PR. As I said, I like the idea, I just think it would be a good practice to discuss it before introducing it. Then we could develop things to incorporate it better like having an attribute in the base class, etc. But no worries about it.